### DISCUSSION TITLE
cursor feels delayed after recent update

### DISCUSSION THREAD

**Discussion by MikkelCornelius** [ppy/osu#29878] [Q&A]
I've had this issue before, where my aim is just off compared to stable. I realized I could fix this by using fullscreen instead of borderless (no idea why it worked). Now with a new update fullscreen no longer works. I can only play in windowed which is less uptimal.

**Comment by Joehuu** [comment]
You are probably experiencing the same problem as https://github.com/ppy/osu/discussions/29846.
